armi/reactor/blocks.py

```diff
@@ -157,7 +157,7 @@ def __deepcopy__(self, memo):
 
         return b
 
-    def _createHomogenizedCopy(self, pinSpatialLocators=False):
+    def createHomogenizedCopy(self, pinSpatialLocators=False):
```

Comment:
This (private) method is referenced in a docstring of one of our internal plugins. Not a huge deal, just FYI.

Reply:
Ah, okay, I'll have to fix that too.

Can you point me at it?  I don't see it.

Reply:
Handled.  Thanks!